docs/docs/javascript/overview/package.mdx

```diff
@@ -65,3 +65,10 @@ Similarly, build scripts can be introduced to Pants via the
 as a way to introduce artifacts generated via bundlers and/or compilers installed and ran via your package manager.
 The result can the be consumed by other targets as either `resource`-targets that can be depended on,
 or as a package for the [docker backend](../../docker/index.mdx).
+
+### Running
+
+Runnable scripts can be specified and invoked through Pants via the `run` goal. This is meant to support scripts
+that are not directly tied to testing or packaging workflows.
```

Comment:
I know you were just following the previous sections wordage, but I think maybe it is better to provide what it is useful for instead, eg `pants run --loop` to show a self-reloading server?

Reply:
I'll merge once this is addressed, thanks!

Reply:
(feel free to ping me if needed)